### ISSUE TITLE
Gesture recognizer delegate

### ISSUE THREAD

**Issue by emptyarray** [colinta/motion-wiretap#6]
I recently ran into a situation where I wanted multiple gesture recognizers on the same element. So I implemented `gestureRecognizer:shouldRecognizeSimultaneouslyWithGestureRecognizer`, but this required me to set all `gestureRecognizers` to have `self` as a `delegate`. I was wondering if MotionWiretap could set the delegate of gesture recognizers to `self` (the `WiretapView`) when created. Do you think this would be a sensible default, or would it be better to let the delegates alone? Thanks.

**Comment by colinta** [comment]
I don't think it would be good to assign `gesture.delegate = self` automatically, but I could see that being an option in the method that creates the gesture:

``` ruby
view.wiretap(:tap, delegate: some_object)
view.wiretap(:tap, delegate: self)  # self is probably a UIViewController
view.wiretap(:tap, delegate: view)
```

Would that accomplish what you're doing?

**Comment by emptyarray** [comment]
Looks good to me!

**Comment by colinta** [comment]
Done, this is available in 1.1.7
